# Request
happypose_ros1/ObjectRequestInfo[] objs
---
# Result
mbot_perception_msgs/RecognizedObject3DList obj_pose
---
# Feedback
string current_process